src/lib/config.js

```diff
@@ -32,29 +32,33 @@ async function config() {
     return;
   }
 
-  // Build table of existing networks found in their config.json
+  // Checks if developer has the legacy networks or deploy aliases in config.json
+  if (!Object.prototype.hasOwnProperty.call(config, 'deployAliases'))
```

Comment:
Does `config.hasOwnProperty('deployAliases')` not work?

Reply:
It does work but this triggers the `no-prototype-builtins` linting error which [disallows calling some Object.prototype methods directly on object instances](https://eslint.org/docs/latest/rules/no-prototype-builtins). We can disable this lint rule or have the code above.

Reply:
Interesting. Thanks for sharing! Good as is